tests_integration/path-serializer.js

```diff
@@ -1,9 +1,18 @@
 "use strict";
 
+const replaceCWD = text => {
+  const cwd = process.cwd();
+  while (text.indexOf(cwd) !== -1) {
+    text = text.replace(cwd, "<cwd>");
+  }
+
+  return text;
+};
```

Comment:
Bad idea, better output absolute path, because you can run prettier using Node API and change cwd so developer do not know cwd

Reply:
Original implemention is replace to `<cwd>`, I only change this, because there might be two path in message.

this is for test, absolute path can't match snap

Reply:
My mistake, sorry